README-translated/README-Spanish.md

```diff
@@ -1,83 +1,120 @@
+| **Cita** |  [![Cite this using DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3233853.svg)](https://doi.org/10.5281/zenodo.3233853) |
+
+*Bienvenidos al repositorio de Github del proyecto The Turing Way. Aquí es donde todos los componentes del proyecto se desarrollan, se revisan y se mantienen.*
 
 _The Turing Way_ es una guía asertiva para la ciencia de datos reproducible. 
-Puedes leerla aquí:: <https://the-turing-way.netlify.com>
 Actualmente estás en el repositorio de GitHub del proyecto donde se encuentran todos los fragmentos que componen la guía y donde ocurre el proceso de escritura/construcción de la misma.
-
 Nuestro objetivo es proporcionar toda la información que las personas que hacen investigación necesitan al inicio de sus proyectos para garantizar que sean fáciles de reproducir al final.
 
-Esto también significa asegurarse de que estudiantes de doctorado, post-doctorantes, investigadoras e investigadores principales y los equipos de financiación sepan qué partes de la "responsabilidad de reproducibilidad" poseen y qué deben hacer para impulsar que la ciencia de datos sean más eficiente, efectiva y comprensible.
+[![El proyecto The Turing Way es un libro, una comunidad, un proyecto de código abierto y una cultura de colaboración. Esto se muestra en cuatro ilustraciones, la primera mostrando el libro de Turing Way, la segunda cómo la comunidad puede crecer, la tercera mostrando dos personas colaborando en una pull request, y la última mostrando el balance donde la reproducibilidad es valorada en mayor medida que el número de documentos publicados](../book/website/figures/README_imgs/README_turingway.png)](https://docs.google.com/presentation/d/13Nm8LcRW87ffxEugGEs5j6HKQEvhiuH8b7MfIdX7MpI/edit#slide=id.p1).
+
+*The Turing Way es un libro, una comunidad y una colaboración global.*
 
-Tabla de contenidos:
+Todas las partes interesadas, incluyendo estudiantes, investigadores, ingenieros de software, líderes de proyectos y equipos fundadores, están animados a usar The Turing Way para entender sus roles y sus responsabilidades en la reproducibilidad de la ciencia de datos.
```

Comment:
```suggestion
Todas las partes interesadas, incluyendo estudiantes, investigadores, ingenieros de software, líderes de proyectos y equipos fundadores, están invitadas a usar The Turing Way para entender sus roles y sus responsabilidades en la reproducibilidad de la ciencia de datos.
```

Hace referencia a las "partes interesadas", así que sería en femenino "interesadas"

Reply:
Solved in 4a14427d83d47024a7d422e652c3ebe0664e216e